client/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 2.1.0 - 2024.01.09
+
+ - Added definitions and diagnostics providers for embedded python and bash
```

Comment:
we can make this better in clarity by attaching the corresponding PR to each of these points and separating the new features and bug fixes. What do you think?

Reply:
Github can generate that for us but it's far less synthetic (more than 30 PRs for this release). Since this list will be presented to users, I prefer this concise summary I wrote rather than a bloated PR list.